### ISSUE TITLE
Error: Error in migration engine. Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs:33:9] internal error: entered unreachable code: render_alter_enum on MySQL 

### ISSUE THREAD

**Issue by divyenduz** [prisma/migrate#580]
Command: `prisma migrate save --experimental -c`
Version: `2.6.0`
Binary Version: `650b5d0348ec38ae61e1e7db69bb54808418ede4`
Report: https://prisma-errors.netlify.app/report/11078 
OS: `x64 darwin 19.6.0`

JS Stacktrace:
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs:33:9] internal error: entered unreachable code: render_alter_enum on MySQL

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

    at ChildProcess.<anonymous> (/Users/danbi/Dev/danbi-api/node_modules/@prisma/cli/build/index.js:2:2196524)
    at ChildProcess.emit (events.js:210:5)
    at ChildProcess.EventEmitter.emit (domain.js:475:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
```

Rust Stacktrace:
```
[migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs:33:9] internal error: entered unreachable code: render_alter_enum on MySQL
```

**Comment by tomhoule** [comment]
This is indeed a bug, I'll take it.

**Comment by tomhoule** [comment]
I wrote extra tests in the engines for enum modifications, although they are already covered, and confirmed that they work: https://github.com/prisma/prisma-engines/pull/1100

I also fiddled around manually with the schema from the error report and the latest dev release, and could not reproduce the crash. I think we can close this issue once https://github.com/prisma/prisma-engines/pull/1100 is merged.
